### ISSUE TITLE
Some issue about ICL Experience

### ISSUE THREAD

**Issue by broalantaps** [princeton-nlp/AutoCompressors#21]
This is an impressive work, thank you for your contribution. However, I have some confusion about the ICL experiments. Did you use the ICL training dataset when fine-tune AutoCompressor from Llama-7B? Also, how did you handle texts longer than 150 characters for datasets like BoolQ and MultiRC? Thank you!

**Comment by anirudhajith** [comment]
Hi @broalantaps! 
No, we did not use any ICL data while fine-tuning any of the AutoCompressor models. 

Regarding your second question, I suspect that you are referring to Table 4's caption in our paper.
> ...note that for BoolQ and MultiRC, demonstrations are too long to fit into 150 tokens.

For datasets like BoolQ and MultiRC were even individual demonstrations exceeded 150 tokens in length, we simply conducted a zero-shot evaluation where the AutoCompressor was prompted to answer the test example expressed as plaintext. 

I hope this answers your questions!
